tests/gcp/test_fg.py

```diff
@@ -1,14 +1,19 @@
 from __future__ import annotations
 
+from inspect import Parameter, signature
+from math import exp, log, pi
+
```

Comment:
I think all these imports are also a bad merge. They don't appear to be used. I don't have pylint setup on our tests or an auto-unused import pruning check setup.